test/pkg/environment/setup.go

```diff
@@ -209,7 +210,7 @@ func (env *Environment) startNodeMonitor(stop <-chan struct{}) {
 	factory.Start(stop)
 }
 
-func (env *Environment) AfterEach() {
+func (env *Environment) AfterEach(excludeObjs ...client.Object) {
```

Comment:
Maybe worth passing in an options object env.Exclude(objects...) to make this interface more apparent. Right now, the caller can't really tell what this means.

Reply:
Sounds good 👍